test/integration/prepare.go

```diff
@@ -139,6 +141,52 @@ func PrepareExampleCC(sdk *fabsdk.FabricSDK, user fabsdk.ContextOption, orgName
 	return nil
 }
 
+// PrepareExampleCCLc install and instantiate using resource management client
+func PrepareExampleCCLc(sdk *fabsdk.FabricSDK, user fabsdk.ContextOption, orgName string, chaincodeID string) error {
+	const (
+		channelID = defaultChannelID
+	)
+
+	fmt.Printf("Installing and instantiating example chaincode..., cc = %v\n", chaincodeID)
+	start := time.Now()
+
+	ccPolicy, err := prepareOneOrgPolicy(sdk, orgName)
+	if err != nil {
+		return errors.WithMessage(err, "CC policy could not be prepared")
+	}
+
+	orgContexts, err := prepareOrgContexts(sdk, user, []string{orgName})
+	if err != nil {
+		return errors.WithMessage(err, "Org contexts could not be prepared")
+	}
+
+	packageID, err := InstallExampleChaincodeLc(orgContexts, chaincodeID, exampleCCVersion)
+	if err != nil {
+		return errors.WithMessage(err, "Installing example chaincode failed")
+	}
+
+	err = ApproveExampleChaincode(orgContexts, channelID, chaincodeID, exampleCCVersion, packageID, ccPolicy, 1)
+	if err != nil {
+		return errors.WithMessage(err, "Approve example chaincode failed")
+	}
+
+	err = CommitExampleChaincode(orgContexts, channelID, chaincodeID, exampleCCVersion, ccPolicy, 1)
```

Comment:
Note, even after the chaincoded fix, I was seeing this error in the CI:

panic: PrepareExampleCCLc return error: Commit example chaincode failed: commit example chaincode failed: sending commit transaction proposal failed: Transaction processing for endorser [peer1.org1.example.com:7151]: Chaincode status Code: (500) UNKNOWN. Description: failed to invoke backing implementation of 'CommitChaincodeDefinition': chaincode definition not agreed to by this org (Org1MSP)

@huangjiehua @bstasyszyn